siliconcompiler/scheduler/slurm.py

```diff
@@ -240,3 +241,13 @@ def run(self):
             if os.path.exists(log_file):
                 self.logger.error(f"Node log file: {log_file}")
             self.halt()
+        
+        # Wait for manifest to propagate through network filesystem
+        start = time.time()
+        elapsed = 0
+        while not os.path.exists(self.get_manifest()) and elapsed <= 2:
+            elapsed = time.time() - start
+            time.sleep(0.5)
```

Comment:
You will need to call: listdir or scandir to trigger the NFS cache from getting updated (this is what we do in several places (including right above this):
https://github.com/Ciprian167/siliconcompiler/blob/e8053f7feec6287a28ad4c4ceb81fe4c839c2cb2/siliconcompiler/scheduler/slurm.py#L224

Could you move the magic number to class const members like `_MAX_FS_DELAY = 2` and `_FS_DWELL = 0.5` you can probably lower the sleep to 0.1 or 0.2 since this would only cause 10 to 20 IO calls.